### PR TITLE
OpenVi 7.1.17 (new formula)

### PR DESCRIPTION
**Pull request by johnsonjh**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Adds [OpenVi](https://github.com/johnsonjh/OpenVi/): Portable OpenBSD vi for UNIX systems.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]

File: Formula/openvi.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  desc "Portable OpenBSD vi for UNIX systems"
+  homepage "https://github.com/johnsonjh/OpenVi#readme"
+  url "https://github.com/johnsonjh/OpenVi/archive/refs/tags/7.1.17.tar.gz"
+  sha256 "02974603f1cd3ba5727fd40f69f97925510d8f30c1ff3041adbc3e54e11f6d5f"
+  license "BSD-3-Clause"
+
+  uses_from_macos "ncurses"
+
+  def install
+    system "make", "install", "CHOWN=true", "LTO=1", "PREFIX=#{prefix}"
```
Comment (SMillerDev) [inline Formula/openvi.rb:11]: What does `CHOWN` do here?

Comment (carlocab) [reply Formula/openvi.rb:11]: `CHOWN=true` probably replaces every call to `chown` with `true`, making it do nothing.

Comment (johnsonjh) [reply Formula/openvi.rb:11]: Yes - the `make install` phase will try to chown the executable during installation (to root).  Not necessary here.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**